lib/hwkey_crypt/hwkey_crypt.c

```diff
@@ -27,13 +27,19 @@
 #include <openssl/evp.h>
 #include <openssl/mem.h>
 
-#include "trusty_key_migration.h"
 #include "trusty_key_crypt.h"
 
 #define LOG_TAG "libhwkey_crypt"
 #define TLOGE(fmt, ...) \
     fprintf(stderr, "%s: %d: " fmt, LOG_TAG, __LINE__,  ## __VA_ARGS__)
 
+static void * (* const volatile memset_ptr)(void *, int, size_t) = memset;
+
+static void secure_memzero(void * p, size_t len)
```

Comment:
you name it "secure_memzero", but how do you make sure memset would be a "secure" memzero?

do you use compiler flag to disable optimization for "memset"?